picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -480,6 +492,54 @@ public Builder transform(@NonNull List<? extends Transformation> transformations
       return this;
     }
 
+    /**
+     * Specifies the {@link MemoryPolicy} to use for this request. You may specify additional policy
+     * options using the varargs parameter.
+     */
+    Builder memoryPolicy(@NonNull MemoryPolicy policy,
+        @NonNull MemoryPolicy... additional) {
+      if (policy == null) {
+        throw new IllegalArgumentException("Memory policy cannot be null.");
```

Comment:
"policy == null" ?

Reply:
Also NPE not IAE

Reply:
Yeah, I can fix these. Just existing code I pasted.